test/it/uk/gov/hmrc/apinotificationpull/acceptance/RetrieveAndDeleteNotificationSpec.scala

```diff
@@ -26,9 +26,9 @@ import play.api.test.Helpers._
 
 class RetrieveAndDeleteNotificationSpec extends FeatureSpec with GivenWhenThen with Matchers with GuiceOneAppPerTest {
 
-  feature("Retrieve a single message from the Pull Service (DELETE)") {
+  feature("Retrieve(DELETE) a single message from the API Notification Pull service") {
```

Comment:
Would the cleanup of this file be better on a separate commit on the `API-2932` branch? These tests relate to that ticket, and could cause a merge conflict, as there are a lot more changes to this spec on that branch

Reply:
Fair enough
I will move this to your branch